### PR TITLE
Support for multi column columns

### PR DESCRIPTION
**Pull request by acuthbert**
Refactored getStorageColumn to createStorageColumns (plural) so you can now return
more than one column to store data in.

getTransformIntoRepository and getTransformFromRepository now take the full model
data as arguments.

getTransformIntoRepository can now return an array of data in order to populate the
full complement of storage columns created in createStorageColumns()

### CODE REVIEW

File: src/Models/Model.php [hunk 1]
```diff
@@ -115,7 +111,13 @@ protected function setDefaultValues()
             }
 
             if ($column->defaultValue !== null) {
-                $this[$column->columnName] = $column->defaultValue;
+                $defaultValue = $column->defaultValue;
+
+                if ( is_object($defaultValue) ){
```
Comment (NiallSmyth) [inline src/Models/Model.php]: Old spacing habits die hard eh? :)

File: src/Models/Model.php [hunk 2]
```diff
@@ -115,7 +111,13 @@ protected function setDefaultValues()
...
+                    $defaultValue = clone( $defaultValue );
```
Comment (NiallSmyth) [inline src/Models/Model.php]: Clone with brackets looks odd - do you have any objection to leaving them off?

Comment (acuthbert) [inline src/Models/Model.php]: Nope - I don't normally do that! I'll remove.

File: src/Models/Model.php (hunk 1 above)
Comment (acuthbert) [inline src/Models/Model.php]: Yeah, finger memory doh!

**Comment by acuthbert** [comment]
Em, okay this is wierd. The PR now thinks that all commits need merged to develop????

**Pull request merged**